packages/Search/src/details/DTK_DetailsKokkosExt.hpp

```diff
@@ -131,6 +132,29 @@ struct infinity<double>
 {
     static constexpr double value = HUGE_VAL;
 };
+
+template <typename T>
```

Comment:
why even define the struct if there is no implementation? (i.e. why not `struct max;` instead of `struct max {};` Is `double` the only implementation we need right now?

Reply:
> why even define the struct if there is no implementation? (i.e. why not `struct max;` instead of `struct max {};`

Good point.  Will change.

> Is `double` the only implementation we need right now?

This is all we were using so far.